api/v1/status_routes.go

```diff
@@ -71,6 +71,10 @@ func HandlePatchStatus(rw http.ResponseWriter, r *http.Request, p httprouter.Par
 		return
 	}
 
+	if status.Stopped {
+		engine.Stop()
```

Comment:
hmm actually, there probably should be a `return` after `engine.Stop()` - it doesn't make sense to pause the test or change the VUs after you have stopped the test :sweat_smile:

Reply:
hmm though you can't just return, since you're also expected to output the status... maybe add the two `if` statements below in in the `else` clause of this `if`?

Reply:
Aha, yes - i had the same argument with myself! :) (I didn't want to repeat the marshalling/error checking at the end and figured if we've signalled the engine to shutdown, it doesn't matter what else has been requested in the PATCH)
I've moved the two `if`'s into an `else`...